test/user.test.js

```diff
@@ -205,6 +205,73 @@ describe('User', function() {
       });
     });
 
+    describe('Password validation with Maxlength', function() {
+      var pass72 = 'ThePasswordShouldNotExceedtheMaximumNumberOfcharactersAndItisInvalidIfIt';
+      var pass73 = 'ThePasswordShouldNotExceedtheMaximumNumberOfcharactersAndItisInvalidIfIt1';
+      var badPass = 'ThePasswordShouldNotExceedtheMaximumNumberOfcharactersAndItisInvalidIfItXYZ1';
+
+      it('rejects passwords longer than 72 characters', function(done) {
+        try {
+          User.create({ email: 'b@c.com', password: pass73 }, function(err) {
+            if (err) return done(err);
+            assert.equal(err.statusCode, 422);
+            assert(false, 'Error should have been thrown');
+          });
+        } catch (e) {
+          done();
+        }
```

Comment:
Yes, I want to ensure that `User.Create` will throw the error, not `User.validatePassword` because the user should be blocked from even creating a user when the password is not valid.